### DISCUSSION TITLE
Bug: Inconsistent date of latest commit

### DISCUSSION THREAD

**Discussion by aqeelat** [community/community#39448] [Repositories]
### Select Topic Area

Bug

### Body

This is a silly bug but it might be a good first issue for your github interns

The header says 8 days
The row says 9 days

<img width="1440" alt="Screenshot 2022-11-18 at 8 50 54 PM" src="https://user-images.githubusercontent.com/5212744/202770187-93f1ca67-6283-43b3-966b-91424d20f523.png">

**Comment by turabek-cc** [comment]
Similarly, where the actual date is December 15 and the last commit date is December 4, it shows as "last month".

![image](https://user-images.githubusercontent.com/104047902/207745334-2ef3c957-949f-4fd5-96aa-60bd0e1ec6ec.png)
